app/router.go

```diff
@@ -76,6 +77,11 @@ func Router(c collector) *mux.Router {
 	return router
 }
 
+func badRequest(r *http.Request, w http.ResponseWriter, err error) {
+	http.Error(w, err.Error(), http.StatusBadRequest)
+	log.Printf("Error procressing request for %s: %v", r.URL.Path, err)
+}
```

Comment:
[Rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) dictates this shouldn't be factored out yet :)